### PR TITLE
[fix](inverted index) Modify Error Handling for File Open Failure

### PR DESCRIPTION
**Pull request by zzzxl1993**
### What problem does this PR solve?

Issue Number: close #xxx

Related PR: #xxx

Problem Summary:

### Release note

None

### Check List (For Author)

- Test 
    - [x] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [x] No.
    - [ ] Yes. 

- Does this need documentation?
    - [x] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

File: regression-test/suites/fault_injection_p0/test_skip_index_compaction_fault_injection.groovy [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+    }
+
+    int rowsetCount = get_rowset_count.call(tablets);
+    assert (rowsetCount == 11 * replicaNum)
+
+    try {
+      // first
+      trigger_full_compaction_on_tablets.call(tablets)
+      wait_full_compaction_done.call(tablets)
+    } finally {
```
Comment (qidaye) [inline regression-test/suites/fault_injection_p0/test_skip_index_compaction_fault_injection.groovy:184]: The first time trigger full compaction will fail. 
Need check rowset count to confirm.

**Review by airborne12 — DISMISSED** [review]
LGTM

**Comment by zzzxl1993** [comment]
run buildall

**Review by qidaye — DISMISSED** [review]
LGTM

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

**Review by airborne12 — APPROVED** [review]
LGTM

**Review by qidaye — APPROVED** [review]
LGTM

**Pull request merged**